### PR TITLE
Readme changes to update the repo/VS Code Marketplace entry

### PR DESCRIPTION
**Pull request by kschaab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,53 @@
...
+
+## Manage your secrets with Secrets Manager
+
+With Cloud Code get the convenience of managing your application secrets directly in VS Code. Quickly connect to
+Google Cloud when running in your IDE locally even while debugging or testing your Kubernetes and Cloud Run
+applications.
+
+![Secret Manager][15]
+
 ## Key Features
```
Comment (russwolf) [inline README.md:32]: missing buildpacks

Comment (kschaab) [reply README.md:32]: How about

## Use Google Cloud buildpacks

With Cloud Code you can focus on building your application, not containerizing it by using Google Cloud buildpacks.
Easily deploy your code directly to GKE or Cloud Run without worrying about a Dockerfile.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,53 @@
...
 
 - Support for Go, Node, Java, and Python
 - Integrated Debugging and Log Viewing/Streaming
 - Cluster and service inspection with the Kubernetes and Cloud Run explorers
 - Cluster creation supporting Google GKE, Amazon EKS and Azure AKS
-- Rapid Edit, Package, Deploy loop to your K8s cluster
+- Rapid Edit, Package, Deploy loop to your Kubernetes and Cloud Run projects
 - Snippets, completions, and linting for K8s artifacts
```
Comment (russwolf) [inline README.md:44]: Should we expand this and the next line to include inline documentation and colorization?

File: README.md [hunk 3]
```diff
@@ -1,34 +1,53 @@
...
 - Support for Custom Resources (CRDs) e.g. Istio, Knative
```
Comment (russwolf) [inline README.md:45]: It's unclear what "Support for Custom Resources" includes. Should we expand/merge this with the above line to call out authoring support (linting, inline docs, completions, colorization) for hundreds of kubernetes CRDs in addition to k8s yaml files

File: README.md [hunk 4]
```diff
@@ -1,34 +1,53 @@
...
-- Automatically uses your Google Cloud SDK credentials
+- Automatically uses your Google Cloud SDK credentials even when using Kubernetes and Cloud Run locally
+- Easily find and manage Google Cloud APIs using the Library Explorer
```
Comment (russwolf) [inline README.md:47]: Should this be "Easily add Google Cloud APIs to your application using the Cloud APIs Explorer" 

could also go to 

"Easily add Google Cloud APIs, such as the Cloud Translation API, to your application using the Cloud APIs Explorer" 

Make it more about the user objective and the problem it solves for them

File: README.md [hunk 5]
```diff
@@ -1,34 +1,53 @@
...
+## Google Cloud API library browser
+
+Cloud Code allows you to find Google Cloud APIs without leaving your IDE and breaking your development flow. Easily
+add these APIs to your project and discover more with our API library browser.
+
+![Client Library Explorer][14]
+
+## Manage your secrets with Secrets Manager
+
+With Cloud Code get the convenience of managing your application secrets directly in VS Code. Quickly connect to
```
Comment (russwolf) [inline README.md:26]: need to add a comma "With Cloud Code, get"

**Pull request merged**